packages/attune/tests/it/main.rs

```diff
@@ -18,3 +31,333 @@ async fn migrations_applied(pool: sqlx::PgPool) {
         "attune_tenant table should exist after migrations"
     );
 }
+
+/// Simulate a `docker compose up` to check that the control plane comes up
+/// properly with all Docker services. Then use the CLI to add many packages
+/// concurrently. Then start a Debian container and `apt install` the packages
+/// to show that the repository works.
+#[tokio::test(flavor = "multi_thread")]
+async fn e2e() {
+    // Initialize tracing.
+    tracing_subscriber::registry()
+        .with(
+            tracing_subscriber::fmt::layer()
+                .with_span_events(FmtSpan::NEW | FmtSpan::CLOSE)
+                .with_file(true)
+                .with_line_number(true)
+                .with_target(true)
+                .with_thread_ids(true)
+                .with_thread_names(true)
+                .with_writer(std::io::stderr)
+                .pretty(),
+        )
+        .with(tracing_subscriber::EnvFilter::from_default_env())
+        .init();
```

Comment:
[optional] consider adding a comment explaining why we aren't just using `test_log` (I assume so we can format more nicely)